hw1/01-fizzbuzz.js

```diff
@@ -7,13 +7,23 @@ Write a program that writes all the numbers from 1 to 100, with some exceptions:
 
 Use console.log() to write the proper output to the command line.
 
-**/
+* */
 
 const fizzbuzz = () => {
-  // Add your code here
-};
+    for (let i = 1; i <= 100; i += 1) {
+        let output = i
+        if (i % 3 === 0 && i % 5 === 0) {
+            output = 'fizzbuzz'
```

Comment:
did a Replace All for backticks - forgot to replace them after experimenting with templates :o)